src/link/rgblink.1

```diff
@@ -89,6 +90,14 @@ Has no effect if
 .Fl O
 is specified.
 The default is 0.
+.It Fl S Ar spec , Fl Fl scramble Ar spec
+Enables use a different
```

Comment:
```suggestion
Enables a different
```
(or "Enables using a different", or just "Enables use of a different", but that's not really necessary.)

Reply:
Yeah, "Enables a different" (my bad); it conveys better that it *selects* that other algorithm, rather than just making it available.